server/cmd/mmctl/commands/auth.go

```diff
@@ -123,7 +126,7 @@ func init() {
 	RootCmd.AddCommand(AuthCmd)
 }
 
-func loginCmdF(cmd *cobra.Command, args []string) error {
+func loginCmdF(c client.Client, cmd *cobra.Command, args []string) error {
```

Comment:
Do we need the `client.Client` any longer?

Reply:
I guess not but at some point I needed it

Reply:
Please remove it then :wink:

Reply:
Removed :heavy_check_mark: